doc/howto/instances_routed_nic_vm.md

```diff
@@ -0,0 +1,57 @@
+(instances-routed-nic-vm)=
+# How to add a routed NIC device to a virtual machine
+
+When adding a {ref}`routed NIC device <nic-routed>` to an instance, you must configure the instance to use the link-local gateway IPs as default routes.
+For containers, this is configured for you automatically.
+For virtual machines, the gateways must be configured manually or via a mechanism like `cloud-init`.
+
+To configure the gateways with `cloud-init`, firstly initialize an instance:
+
+    lxc init ubuntu:jammy jammy --vm
```

Comment:
I think we use ubuntu:22.04 in rest of docs, please can you check if that is so and if it is align with that. Ta

Reply:
I've changed to `22.04` now.

Reply:
Thanks, I missed that!